joint_state_controller:
  type: effort_controllers/JointTrajectoryController
  publish_rate: 50

FL_joint_controller:
  type: effort_controllers/JointPositionController
  joints:
    - FLS_joint
    - FLL_joint
    - FLF_joint
  gains:
    FLS_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
    FLL_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
    FLF_joint: {p: 100, d: 1, i: 1, i_clamp: 1}
